tests/integration/ceph_csi_test.go

```diff
@@ -128,6 +165,19 @@ parameters:
 	require.Nil(s.T(), err)
 }
 
+func deleteCSIStorageClass(k8sh *utils.K8sHelper, namespace string) {
+	err := k8sh.Clientset.StorageV1().StorageClasses().Delete(csiSCRBD, &metav1.DeleteOptions{})
+	if err != nil {
+		logger.Errorf("failed to delete rbd storage class %s with error %v", csiSCRBD, err)
+	}
+	err = k8sh.Clientset.StorageV1().StorageClasses().Delete(csiSCCephFS, &metav1.DeleteOptions{})
+	if err != nil {
+		logger.Errorf("failed to delete cephfs storage class %s with error %v", csiSCCephFS, err)
+		return
+	}
+	logger.Info("Deleted rbd and cephfs storageclass")
+}
+
 func createCSIRBDTestPod(k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
```

Comment:
can you change the name of this method since it's both creating and deleting the test pod?